server/apps/prepopulate/app_initialization_test.py

```diff
@@ -1,52 +1,46 @@
-from superdesk.tests import TestCase
+from test_factory import SuperdeskTestCase
 from .app_initialize import AppInitializeWithDataCommand
 from .app_scaffold_data import AppScaffoldDataCommand
 from superdesk import get_resource_service
 
 
-class AppInitializeWithDataCommandTestCase(TestCase):
-
-    def setUp(self):
-        super().setUp()
+class AppInitializeWithDataCommandTestCase(SuperdeskTestCase):
 
     def test_app_initialization(self):
-        with self.app.app_context():
-            command = AppInitializeWithDataCommand()
-            result = command.run()
-            self.assertEquals(result, 0)
+        command = AppInitializeWithDataCommand()
+        result = command.run()
+        self.assertEquals(result, 0)
```

Comment:
No sure if this is the right PR for changing this, `assertEquals()` is deprecated and should be renamed to `assertEqual()`.